tests/language-feature-scripts/lazy-loading-ghost-allows-protected-property-reflection-write.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Verifies that generated access interceptors allow protected property write via Reflection
+--FILE--
+<?php
+
+require_once __DIR__ . '/init.php';
+
+class Kitchen
+{
+    protected $sweets;
+}
+
+$factory = new \ProxyManager\Factory\AccessInterceptorValueHolderFactory($configuration);
```

Comment:
Just noticed that the file name does not match the test case... Indeed, this is a value holder issue, not a ghost object isse